readme.md

```diff
@@ -75,6 +75,13 @@ PR welcome for additional commonly needed types and docs improvements. Read the
 npm install type-fest
 ```
 
+<p>Or, for projects that do not export these or dependent types, install as a dev only dependency with: </p>
```

Comment:
This is not entirely accurate. The types don't need to be exported for it to need to be a dependency, and not dev dependency. Even if the types are used internally in a index.d.ts file (and not exported), `type-fest` needs to a dependency. Basically, if types included in a package use types from `type-fest`, `type-fest` needs to be a dependency.

Reply:
Unless one explicitly add an ignore to the imports, then they become optional imports, see eg this helper of mine: https://github.com/voxpelli/ts-ignore-import